core/src/components/toast/toast.tsx

```diff
@@ -484,6 +493,16 @@ export class Toast implements ComponentInterface, OverlayInterface {
     return buttons;
   }
 
+  private getShape(theme: Theme): string | undefined {
```

Comment:
Theme does not need to be passed. It can be done through `const theme = getIonTheme(this);`